### DISCUSSION TITLE
Can't create an app

### DISCUSSION THREAD

**Discussion by Peppper74** [quasarframework/quasar#9862] [CLI - SSR mode] [answered]
Hi! I am trying to create a new application but I get an error.
`D:\WORK\WebstormProjects>quasar create test
 App • Looking for Quasar App Extension "create" command "test"

 App • ⚠️  Error. This command must be executed inside a Quasar v1+ project folder.
`

**Comment by MilosPaunovic** [comment]
@Peppper74 Could you provide some more info, let's say run `quasar info` command and paste its result here for starters.

**Reply by Autrui** [reply]
When I run quasar info, It shows
Error. This command must be executed inside a Quasar v1+ project folder.

**Comment by pdanpdan** [comment]
Most probably you managed to install `@quasar/app` globally instead of `@quasar/cli`

**Comment by hawkeye64** [comment, marked as answer]
@Autrui-K I think the same as Dan.
```
npm r -g @quasar/app
npm i -g @quasar/cli
```
Hopefully, that fixes your issues

**Reply by Autrui** [reply]
Thanks for your answer,it actually works.

**Reply by diego-yd** [reply]
I have same problem but it doesnt work for me

**Reply by rstoenescu** [reply]
> I have same problem but it doesnt work for me
> 
> 

You must be doing something wrong. Post a minimal reproduction or something that we can look at so we can help.

**Reply by diego-yd** [reply]
Yup . my mistake was i used 'quasor' instead of 'quasar'. I changed it and now it works perfectly,

**Reply by hawkeye64** [reply]
@gaurabbb Are you by chance trying to create a new project that's already inside a mono-repo? I just found out yesterday, creating a quasar project within the parent of a `yarn workspace` did not work for me. I mean, the projects created, but trying to run `quasar dev` resulted in the same errors as above.

**Reply by nateiler** [reply]
@hawkeye64 we're also using a `yarn workspace` mono-repo and seeing the same error.  Did you figure out some sort of work around?

**Reply by hawkeye64** [reply]
The work-around was to create the project outside of the mono-repo, and then when it's done, move it where it needs to go and add it to the mono-repo, and then life is good again. :)

**Reply by rabee05** [reply]
@rstoenescu I have same issue and I just create a simple default project using vite 

To get started:

  cd quasar-project
  quasar dev # or: yarn quasar dev # or: npx quasar dev

Documentation can be found at: https://v2.quasar.dev

Quasar is relying on donations to evolve. We'd be very grateful if you can
read our manifest on "Why donations are important": https://v2.quasar.dev/why-donate
Donation campaign: https://donate.quasar.dev
Any amount is very welcome.
If invoices are required, please first contact Razvan Stoenescu.

Please give us a star on Github if you appreciate our work:
  https://github.com/quasarframework/quasar

Enjoy! - Quasar Team

xx@host:~/temp$ cd quasar-project/
xx@host:~/temp/quasar-project$ ls
index.html  jsconfig.json  node_modules  package.json  package-lock.json  postcss.config.js  public  quasar.config.js  README.md  src
xx@host:~/temp/quasar-project$ quasar dev

 App • ⚠️  Error. This command must be executed inside a Quasar v1+ project folder.

I notice this issue on ubuntu 22.04, nodejs version v16.17.0


however, when I run inside the project 
xx@host:~/temp/quasar-project$ quasar -v
3.0.0-rc.2

This is newly created project

but when I run the same on windows 
PS C:\temp\quasar-project>  quasar.cmd -v 
3.5.9

**Comment by ahmed82** [comment]
Clearing the Global lib  as @hawkeye64  advice. Fix most of the common issues during creating the App
npm r -g @quasar/app
npm i -g @quasar/cli

**Comment by nateiler** [comment]
I was experiencing the same error, but in our scenario we have a mono-repo via `lerna` and `yarn workspaces`.  It looks like new projects are using `@quasar/app-vite` and `@quasar/app-webpack` while older projects are using `@quasar/app`.  After replacing `@quasar/app` w/ `@quasar/app-webpack` (the old projects are webpack) the build error stopped.

TLDR: Make sure all of your quasar projects `@quasar/app-vite` or `@quasar/app-webpack` and not `@quasar/app`

**Reply by hawkeye64** [reply]
running `quasar upgrade -i` should do this for you.
